app/javascript/hyacinth_v1/components/projects/enabled_dynamic_fields/EnabledDynamicFieldForm.js

```diff
@@ -223,7 +223,10 @@ export const EnabledDynamicFieldForm = (props) => {
   const [disabled] = useState(formType !== 'edit');
   const [enabledDynamicFields] = useState({});
 
-  const variables = { project: { stringKey: projectStringKey }, digitalObjectType };
+  // ALL CAPS enum value per graphql API
+  const digitalObjectTypeUcase = digitalObjectType.toUpperCase();
```

Comment:
Same thought here as earlier comment about expecting digitalObjectType to come back from the graphql api as upper case.  (Sorry for repetitive comment.  Just wanted to note the places where I saw this.)